modules/statics/src/coins.ts

```diff
@@ -14082,6 +14084,22 @@ export const coins = CoinMap.fromCoins([
     '0xe9df68a54bba438c8a6192e95f0f2c53ac93d997',
     UnderlyingAsset['topeth:terc18dp']
   ),
+  zkethErc20(
+    '53f0e845-f415-44d3-8517-7565dc346390',
+    'zketh:link',
+    'Chainlink Token',
```

Comment:
why chainlink token ??

Reply:
This is to add support for erc20 tokens on zkSync. And to start off with Chainllink. Kind of like onboarding this erc20 token.